src/title.ts

```diff
@@ -16,10 +21,17 @@ export interface TitleBase {
    */
   offset?: number;
 
+  /**
+   * A [mark style property](config.html#style) to apply to the title text mark.
+   *
+   * __Default value:__ `"group-title"`.
+   */
+  style?: string | string[];
+
   // TODO: name, encode, interactive, zindex
 }
 
-export interface Title extends TitleBase {
+export interface TitleParams extends TitleBase {
```

Comment:
We use `bin: boolean | BinParams` so that in the documentation it shows up as Bin is either a boolean or a BinParams object (not boolean or Bin, because boolean is Bin too (Bin = boolean | BinParams).

So I'm doing the same thing here.  (Title = string | TitleParams) but we don't define Title explicitly because we want both types to show up in the table in the docs.